integration-test/src/main/java/com/sequenceiq/it/cloudbreak/newway/cloud/AwsCloudProvider.java

```diff
@@ -264,6 +268,41 @@ public NetworkV2Request existingSubnet() {
         return map;
     }
 
+    @Override
+    public Set<RDSConfigRequest> getRdsRequestsFor(RdsType... rdsTypes) {
+        Set<RDSConfigRequest> requests = new LinkedHashSet<>();
+        if (rdsTypes != null) {
+            for (RdsType provider : rdsTypes) {
+                switch (provider) {
```

Comment:
What is the problem with 
 request.add(createRdsRequestWithProperties(getTestParameter(), **provider**, AWS_CAPITAL));